seed/challenges/basic-javascript.json

```diff
@@ -176,7 +175,7 @@
         "Try looking at the <code>secondLetterOfFirstName</code> variable declaration if you get stuck."
       ],
       "tests": [
-        "assert(thirdLetterOfLastName === 'v', 'message: The third letter of <code>lastName</code> should be a \"v\".');"
+        "assert(thirdLetterOfLastName === 'v', 'message: The <code>thirdLetterOfLastName</code> variable should have the value of \"v\".');"
```

Comment:
If you're going to be consistent, we should wrap `v` with code tags.   I would omit the quotes both here and above for `L`.   Or remove the code tags and keep the quotes..